doc/user-guide/How-to-build.md

```diff
@@ -82,32 +127,41 @@ To run MongooseIM from the project tree after compiling it, change to `$REPO/_bu
 There you can use the `mongooseim` command line administration script to start and stop MongooseIM.
 For example, this command will start the server:
 
-    $ bin/mongooseim start
+```bash
+$ bin/mongooseim start
+```
 
 You can also run the server in interactive mode (drop into an Erlang shell):
 
-    $ bin/mongooseim live
+```bash
+$ bin/mongooseim live
+```
 
 There's also a tool called `mongooseimctl` to perform some operations on a running instance, e.g.:
 
-    $ bin/mongooseimctl status
-    MongooseIM node mongooseim@localhost:
-      operating system pid: 86026
-      Erlang VM status: started (of: starting | started | stopping)
-      boot script status: started
-      version: 1.6.2-61-g48b8332
-      uptime: 1:12:46
-      logs:
+```
+$ bin/mongooseimctl status
+MongooseIM node mongooseim@localhost:
+    operating system pid: 3105
+    Erlang VM status: started (of: starting | started | stopping)
+    boot script status: started
+    version: 3.4.0-7-gaec944c92 (as mongooseim)
+    uptime: 0 days 00:00:12
+    distribution protocol: inet_tcp
+    logs:
         log/ejabberd.log
+```
+
 
 ## Building the testing target and running tests
 
 For testing purposes there's a different make target available:
```

Comment:
Hey, I don't understand 'make target' is that a developer specific term everyone will understand?

Reply:
@AnthonyJeannot yeap, `make` is a developer tool, every developer knows it.